jormungandr/src/main.rs

```diff
@@ -101,6 +101,7 @@ fn start() -> Result<(), start_up::Error> {
 pub struct BootstrappedNode {
     settings: Settings,
     blockchain: BlockchainR,
+    block0_hash: HeaderHash,
```

Comment:
This information should be already present in `BlockchainR`, in tip ledger static params, so it seems like redundancy

Reply:
not the same data. This is for the **Block 0**. The network needs to know what _node0_ we are talking about in order to be able to detect if a node is working on a different blockchain. (testnet vs mainnet for example)